building/tracks/concept-map.md

```diff
@@ -0,0 +1,48 @@
+# Concept Map
+
+The concept map is one of the main methods to illustrate the progression that a student takes through the concept exercises.
+
+## Design Goals
+
+- Provide a meaningful concept map of ideas from simple to complex ideas.
+- Provide a pathway towards fluency in a language.
+- Illustrate the progression through the course [track] contents.
```

Comment:
What does `[track]` refer to?

Reply:
I had added that as a colloquialism, but probably isn't necessary

Reply:
```suggestion
- Illustrate the progression through the course contents.
```